### PR TITLE
[Truffle] Add CExt test for ruby-argon2

### PR DESCRIPTION
**Pull request by lxp**
Depends on https://github.com/jruby/jruby-truffle-gem-test-pack/pull/1

### CODE REVIEW

File: tool/jt.rb [hunk 1]
```diff
@@ -838,17 +847,20 @@ def test_cexts(*args)
     [
         ['oily_png', ['chunky_png-1.3.6', 'oily_png-1.2.0'], ['oily_png']],
         ['psd_native', ['chunky_png-1.3.6', 'oily_png-1.2.0', 'bindata-2.3.1', 'hashie-3.4.4', 'psd-enginedata-1.1.1', 'psd-2.1.2', 'psd_native-1.1.3'], ['oily_png', 'psd_native']],
-        ['nokogiri', [], ['nokogiri']]
-    ].each do |gem_name, dependencies, libs|
+        ['nokogiri', [], ['nokogiri']],
+        ['ruby-argon2', [], ['ruby-argon2'], {'GEM_HOME' => "#{ENV['GEM_HOME']}/bundler/gems/ruby-argon2-afe6d65e37c6", 'TEST_CHECKS' => '1'}]
```
Comment (eregon) [inline tool/jt.rb]: Should GEM_HOME be set to a specific gem like that?

Comment (eregon) [inline tool/jt.rb]: I fixed `jt cextc` so this should no longer be needed only
` ['ruby-argon2', [], ['ruby-argon2']],`.
For `TEST_CHECK`, maybe the default can be `1` in the test harness?
Configuration is always easier to keep closer to the source :)

Comment (eregon) [inline tool/jt.rb]: We need a way to specify the path to the argon2 dir in the test pack though, probably can be an extra argument in this list.

**Comment by pitr-ch** [comment]
jruby/jruby-truffle-gem-test-pack#1 looks good

**Pull request merged**